bin/test/errors/dune

```diff
@@ -0,0 +1,2 @@
+(cram
+ (deps %{bin:mustache}))
```

Comment:
You should depend on all of the .json and .mustache files as well. Or initialize them as part of the test.